src/Umbraco.Web.UI.Client/src/views/dashboard/settings/healthcheck.html

```diff
@@ -14,6 +14,7 @@
                         <umb-button type="button"
                                     button-style="success"
                                     label="Check All Groups"
+                                    data-element="healthchecks-checkall-btn"
```

Comment:
Thanks for this new test  @Bakersbakebread ! 👍

I was just wondering whether the data-element attribute was needed here, because it does not seem to be used in any locator/selector in the playwright script. Or am I overseeing something?

Cheers!

Reply:
It's not, but I added it initially for consistency with other buttons!

Reply:
That's a very valid reason then 😁

Let's merge this! Thanks again @Bakersbakebread for improving the test base 🎉